frontend/src/lib/i18n/index.js

```diff
@@ -1,15 +1,21 @@
-import { browser } from '$app/environment';
+import {} from '$app/environment';
 import { init, register } from 'svelte-i18n';
+import { browser } from '$app/environment';
 
 const defaultLocale = 'en';
 
 register('en', () => import('./locales/en.json'));
 register('ru', () => import('./locales/ru.json'));
-if (browser) {
-  console.log({ defaultLocale, navigator: window.navigator.language });
+
+export function localeStoragevalue() {
+  let localevalue = browser ? window.localStorage.getItem('locale') : null;
+  if (localevalue === null) {
+    localevalue = browser ? window.navigator.language : defaultLocale;
+  }
+  return localevalue;
```

Comment:
I would rewrite this logic as follows (note I renamed this function into `defineLocale` because it's better defines the purpose of this function

```js
const savedLocale = browser ? window.localStorage.getItem('locale') : null;

export function defineLocale() {
  if (savedLocale) { return savedLocale; }

  return browser ? window.navigator.language : defaultLocale;
}

```